src/app/app.component.ts

```diff
@@ -5,4 +5,11 @@ import { Component } from '@angular/core';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {}
+export class AppComponent {
+  constructor() {
+    const preload = document.createElement('link');
+    preload.rel = 'preconnect';
+    preload.href = window.location.origin;
```

Comment:
why we are doing this?

Reply:
preconnect for optimized image. Need to look into this. Commenting for now to revisit later

Reply:
Commented the code. Need to revisit to look for another strategy.